superset-frontend/src/chart/ChartRenderer.jsx

```diff
@@ -222,13 +229,18 @@ class ChartRenderer extends React.Component {
       queryResponse,
     } = this.props;
 
+    let chartClassName = snakeCase(vizType);
+    if (isMigratedViz(vizType)) {
+      chartClassName = `superset-chart-${chartClassName}`;
+    }
+
     return (
       <>
         {this.renderTooltip()}
         <SuperChart
           disableErrorBoundary
           id={`chart-id-${chartId}`}
-          className={`${snakeCase(vizType)}`}
+          className={chartClassName}
```

Comment:
This will change (fix) the looks of the table chart.

Previously the whole table has a gray background:
![Snip20200303_4](https://user-images.githubusercontent.com/335541/75828912-c5cd5300-5d61-11ea-90de-1ee451c647c9.png)

which seems to be an unintentional because it comes from a CSS rule `.table .table` for [nested Bootstrap table](https://github.com/twbs/bootstrap/blob/68b0d231a13201eb14acd3dc84e51543d16e5f7e/less/tables.less#L84-L87). 

Removing the background does also give us the strips:

![image](https://user-images.githubusercontent.com/335541/75829481-26a95b00-5d63-11ea-8191-1f8e47bdeb1b.png)

Reply:
ooo, nice catch!